hbjs.js

```diff
@@ -60,10 +60,29 @@ function hbjs(instance) {
     function glyphToPath(glyphId) {
       var svgLength = exports.hbjs_glyph_svg(ptr, glyphId, pathBuffer, pathBufferSize);
       return svgLength > 0 ? utf8Decoder.decode(heapu8.subarray(pathBuffer, pathBuffer + svgLength)) : "";
+    /**
+     * Return glyph name.
+     * @param {number} glyphId ID of the requested glyph in the font.
+     **/
+    function glyphName(glyphId) {
+      var ok = exports.hb_font_get_glyph_name(
+        ptr,
+        glyphId,
+        nameBuffer,
+        nameBufferSize
+      );
+      if (!ok) {
+        return '';
+      }
```

Comment:
You might want to use `hb_font_glyph_to_string()` instead which never fails (generates `gidDDD` names instead), and may be expose `hb_font_glyph_from_string()` for parity.